lib/src/signed_video_h26x_auth.c

```diff
@@ -116,7 +132,25 @@ decode_sei_data(signed_video_t *self, const uint8_t *payload, size_t payload_siz
 }
 
 /**
- * Computes the GOP hash from |nalu_hash_list| and compares it with the GOP hash received from the
+ * Compares the computed link hash with the linked hash received from the
+ * signature.
+ */
+bool
+verify_linked_hash(signed_video_t *self)
+{
+  gop_info_t *gop_info = self->gop_info;
+  const size_t hash_size = self->verify_data->hash_size;
+  // If linked hash validation is not feasible, the linked hash verification
+  // does not affect the final outcome of the validation. Therefore, return true.
```

Comment:
Rewrite comment to something like
"The linked hash is used to verify the order of GOPs. By definition, this is only feasible after having received two complete GOPs. This can be determined by identifying all-zero hashes in |linked_hashes|."